### ISSUE TITLE
flexbox icons

### ISSUE THREAD

**Issue by trungdo-epi** [lucide-icons/lucide#399]
## Icon Request

* Icon name: flexbox
* Use case: page builder
* _Screenshots_ of similar icons:

I take these icons from the codepen https://codepen.io/wstone/pen/mOONLK.

flex direction column
![column](https://user-images.githubusercontent.com/56108514/133207955-959ab934-fd25-478a-b477-2b5d13aa0645.PNG)
flex direction row
![row](https://user-images.githubusercontent.com/56108514/133207976-00509207-a471-498f-a6b5-455d8b3a0177.PNG)

wrap for row
![row-wrap](https://user-images.githubusercontent.com/56108514/133208477-0305fbbf-5841-4a08-a771-16152257a4b3.PNG)
justify content for row
![row-justify](https://user-images.githubusercontent.com/56108514/133208155-06780609-ceb5-4475-aea5-abbec557a86c.PNG)
align items for row
![row-align](https://user-images.githubusercontent.com/56108514/133208188-4153efa0-b7e4-401a-9a2f-eed90743afe5.PNG)

wrap for column
![col-wrap](https://user-images.githubusercontent.com/56108514/133208635-78e4bd65-b9a1-4a1d-b34d-4c0e533bd6ca.PNG)
justify content for column
![col-justify](https://user-images.githubusercontent.com/56108514/133208656-1e822a37-e46e-47c3-ae49-7a97dc1d5a50.PNG)
align items for column
![col-aign](https://user-images.githubusercontent.com/56108514/133208685-571cad8e-11ac-4e51-a9d5-b4bb5df46666.PNG)

Thanks.

**Comment by ericfennis** [comment]
@trungdo-epi Nice request!

**Comment by trungdo-epi** [comment]
@ericfennis Thanks. I am very appreciate lucide team's work.

**Comment by ericfennis** [comment]
@trungdo-epi Allright I this is my result. A lot of icons.
Beside flesbox these type of icons are also really usefull for design applications. So I took some inspiration from illustrator, figma and webflow. Webflow had specific flexbox interface like things.
![image](https://user-images.githubusercontent.com/11825403/144888486-4d5e1fda-ac56-4a3f-9c94-b758025513b4.png)

After I designed this Chromium Devtools release a new interface tool for flexbox. I really like these, it also fits the way I designed these I think. Only missing icons are: flex-direction and flex-wrap icons, plus baseline. But baseline icon can be useful for other things as well.
![image](https://user-images.githubusercontent.com/11825403/144889501-3984cc11-adb8-4cd4-bd16-1379d87f3a39.png)
